_includes/testcase.html

```diff
@@ -18,10 +18,16 @@ <h2>{{ include.testname }}</h2>
   {% endif %}
 
   <div class="testcase">
-    <pre><code>{{ code }}</code></pre>
-    <div data-outcome="{{ result }}" data-outcome-id="{{ testName }}">
-      {{ html }}
-    </div>
+      <article>
+            <header>Code Snippet:</header>
```

Comment:
I don't think the semantics on this are right. Probably best to just use figure:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

Reply:
Updated:

![image](https://user-images.githubusercontent.com/20978252/37599582-fe93ea28-2b7c-11e8-801c-8df44b61248a.png)